<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceInfoWidget</class>
 <widget class="QWidget" name="SourceInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>460</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
   <item>
    <widget class="QLabel" name="sourceLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Source Name</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Recent Albums</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AlbumView" name="recentAlbumView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>192</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>192</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Latest Additions to their Collection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CollectionView" name="recentCollectionView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CollectionView</class>
   <extends>QTreeView</extends>
   <header>collectionview.h</header>
  </customwidget>
  <customwidget>
   <class>AlbumView</class>
   <extends>QListView</extends>
   <header>albumview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
